process/dataValidators/nilTxValidator.go

```diff
@@ -0,0 +1,32 @@
+package dataValidators
+
+import (
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+// nilTxValidator represents a tx handler validator that doesn't check the validity of provided txHandler
+type nilTxValidator struct {
+}
+
+// NewNilTxValidator creates a new nil tx handler validator instance
+func NewNilTxValidator() (*nilTxValidator, error) {
+	return &nilTxValidator{}, nil
+}
+
+// IsTxValidForProcessing is a nil implementation that will return true
+func (ntv *nilTxValidator) IsTxValidForProcessing(interceptedTx process.TxValidatorHandler) bool {
+	return true
+}
+
+// NumRejectedTxs will return number of rejected transaction
+func (ntv *nilTxValidator) NumRejectedTxs() uint64 {
```

Comment:
not tested

Reply:
Will remove all tests as it becomes a mock